src/Storages/StorageReplicatedMergeTree.cpp

```diff
@@ -7656,8 +7656,7 @@ void StorageReplicatedMergeTree::replacePartitionFrom(
             entry_replace.columns_version = -1;
         }
 
-        if (replace)
-        {
```

Comment:
Looks good. Let's revert it and run full checks to see the other reports

Reply:
And there's a lost remaining in report.py: `if (classes.contains("failed-content") || classes.contains("failed-content.open"))`